### PR TITLE
Improve responsiveness

### PR DESCRIPTION
**Pull request by HeDo88TH**
Several UI fixes to improve responsiveness.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,13 @@
...
     "webpack-bundle-tracker": "0.0.93",
-    "webpack-livereload-plugin": "^2.1.1"
+    "webpack-livereload-plugin": "^2.1.1",
+    "autoprefixer": "^9.8.4",
+    "cssnano": "^4.1.10",
+    "gulp": "^4.0.2",
+    "gulp-postcss": "^8.0.0",
+    "gulp-sass": "^4.1.0",
+    "gulp-sourcemaps": "^2.6.5",
+    "node-sass": "^5.0.0"
```
Comment (pierotofy) [inline package.json:78]: Can we not add any of these? We already have a build system (webpack). Why add gulp?

Especially `node-sass`, which requires C++ code compilation and does not reliably build on Windows. https://github.com/OpenDroneMap/WebODM/commit/85b2a8a0db688f30ebbec1fd87b14d25164f5d50

Comment (pierotofy) [reply package.json:78]: If you don't want to setup the build system (simpler), build the static CSS files locally however you want and commit them to the repo. The admin area doesn't get edited very often (it's a built-in component), so it's OK to do that.

Comment (HeDo88TH) [reply package.json]: Removed gulp in https://github.com/OpenDroneMap/WebODM/pull/1129/commits/a325cf1f9858c7606254726197f1f65fbeab81b3, there is a tool in vscode that does it automatically https://marketplace.visualstudio.com/items?itemName=ritwickdey.live-sass

Comment (pierotofy) [reply package.json]: You might have to commit the CSS files back into the repo though.

Comment (HeDo88TH) [reply package.json]: I think they are in the repo already https://github.com/HeDo88TH/WebODM/tree/improve-responsiveness/app/static/admin/css

**Comment by pierotofy** [comment]
1. Expand a task.
2. Witness:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'toFixed')
    at TaskListItem.render (TaskListItem.jsx:694:1838)
    at finishClassComponent (react-dom.development.js:17160:31)
    at updateClassComponent (react-dom.development.js:17110:24)
    at beginWork (react-dom.development.js:18620:16)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237:16)
    at invokeGuardedCallback (react-dom.development.js:292:31)
    at beginWork$1 (react-dom.development.js:23203:7)
    at performUnitOfWork (react-dom.development.js:22154:12)
    at workLoopSync (react-dom.development.js:22130:22)
render @ TaskListItem.jsx:694
finishClassComponent @ react-dom.development.js:17160
updateClassComponent @ react-dom.development.js:17110
beginWork @ react-dom.development.js:18620
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
beginWork$1 @ react-dom.development.js:23203
performUnitOfWork @ react-dom.development.js:22154
workLoopSync @ react-dom.development.js:22130
performSyncWorkOnRoot @ react-dom.development.js:21756
eval @ react-dom.development.js:11089
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
flushSyncCallbackQueueImpl @ react-dom.development.js:11084
flushSyncCallbackQueue @ react-dom.development.js:11072
scheduleUpdateOnFiber @ react-dom.development.js:21199
enqueueSetState @ react-dom.development.js:12639
Component.setState @ react.development.js:471
eval @ TaskList.jsx:98
j @ jquery-1.11.2.min.js:2
fireWith @ jquery-1.11.2.min.js:2
x @ jquery-1.11.2.min.js:4
b @ jquery-1.11.2.min.js:4
XMLHttpRequest.send (async)
send @ jquery-1.11.2.min.js:4
ajax @ jquery-1.11.2.min.js:4
m.<computed> @ jquery-1.11.2.min.js:4
getJSON @ jquery-1.11.2.min.js:4
loadTaskList @ TaskList.jsx:97
componentDidMount @ TaskList.jsx:76
commitLifeCycles @ react-dom.development.js:19814
commitLayoutEffects @ react-dom.development.js:22803
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
commitRootImpl @ react-dom.development.js:22541
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
commitRoot @ react-dom.development.js:22381
finishSyncRender @ react-dom.development.js:21807
performSyncWorkOnRoot @ react-dom.development.js:21793
eval @ react-dom.development.js:11089
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
flushSyncCallbackQueueImpl @ react-dom.development.js:11084
flushSyncCallbackQueue @ react-dom.development.js:11072
scheduleUpdateOnFiber @ react-dom.development.js:21199
enqueueSetState @ react-dom.development.js:12639
Component.setState @ react.development.js:471
eval @ ProjectList.jsx:108
j @ jquery-1.11.2.min.js:2
fireWith @ jquery-1.11.2.min.js:2
x @ jquery-1.11.2.min.js:4
b @ jquery-1.11.2.min.js:4
XMLHttpRequest.send (async)
send @ jquery-1.11.2.min.js:4
ajax @ jquery-1.11.2.min.js:4
m.<computed> @ jquery-1.11.2.min.js:4
getJSON @ jquery-1.11.2.min.js:4
refresh @ ProjectList.jsx:106
componentDidMount @ ProjectList.jsx:88
commitLifeCycles @ react-dom.development.js:19814
commitLayoutEffects @ react-dom.development.js:22803
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
commitRootImpl @ react-dom.development.js:22541
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
commitRoot @ react-dom.development.js:22381
finishSyncRender @ react-dom.development.js:21807
performSyncWorkOnRoot @ react-dom.development.js:21793
scheduleUpdateOnFiber @ react-dom.development.js:21188
updateContainer @ react-dom.development.js:24373
eval @ react-dom.development.js:24758
unbatchedUpdates @ react-dom.development.js:21903
legacyRenderSubtreeIntoContainer @ react-dom.development.js:24757
render @ react-dom.development.js:24840
eval @ Dashboard.jsx:130
each @ jquery-1.11.2.min.js:2
each @ jquery-1.11.2.min.js:2
eval @ Dashboard.jsx:129
j @ jquery-1.11.2.min.js:2
add @ jquery-1.11.2.min.js:2
m.fn.ready @ jquery-1.11.2.min.js:2
m.fn.init @ jquery-1.11.2.min.js:2
m @ jquery-1.11.2.min.js:2
eval @ Dashboard.jsx:128
./app/static/app/js/Dashboard.jsx @ Dashboard-a3491d5a20cef0779f80.js:199
__webpack_require__ @ Dashboard-a3491d5a20cef0779f80.js:20
eval @ Dashboard.jsx:1
2 @ Dashboard-a3491d5a20cef0779f80.js:1847
__webpack_require__ @ Dashboard-a3491d5a20cef0779f80.js:20
(anonymous) @ Dashboard-a3491d5a20cef0779f80.js:95
(anonymous) @ Dashboard-a3491d5a20cef0779f80.js:98
Show 40 more frames
react_devtools_backend.js:4061 The above error occurred in the <TaskListItem> component:
    in TaskListItem
    in div
    in TaskList
    in div
    in li
    in ProjectListItem
    in ul
    in div
    in Paginator
    in div
    in ProjectList
    in projectList
    in Route
    in div
    in Router
    in BrowserRouter
    in Dashboard
```

**Comment by HeDo88TH** [comment]
Fixed! @pierotofy

**Comment by pierotofy** [comment]
Padding is uneven:

image


How come is there so much padding on desktop now? Also, not so sure about the "show task output" button being in the center.

image

**Comment by HeDo88TH** [comment]
Now it looks good 🚀 @pierotofy

**Review by pierotofy — APPROVED** [review]

**Pull request merged**